build/repo.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+   <ItemGroup>
+     <ExcludeFromPack Include="$(RepositoryRoot)src\Microsoft.AspNetCore.AzureAppServices.TestBundle\Microsoft.AspNetCore.AzureAppServices.TestBundle.csproj" />
```

Comment:
Or something like tools/ which I use as a catch all for non-sample, non-src, non-sample things. packaging/ could work too but I think there's less precedent with that naming.